README.md

```diff
@@ -70,6 +70,10 @@ rhp2:
   address: :9982
 rhp3:
   tcp: :9983
+rhp4:
+  listenAddresses:
```

Comment:
Default is two listen addresses:

```yaml
- protocol: tcp
address: :9984
- protocol: quic
address: :9984
```

Other than that small change thanks for catching this.